pkg/controller/wip/controller.go

```diff
@@ -718,3 +750,13 @@ func convertServicePlans(plans []brokerapi.ServicePlan) []v1alpha1.ServicePlan {
 	}
 	return ret
 }
+
+func unmarshalParameters(in []byte) (map[string]interface{}, error) {
```

Comment:
So, we get a raw yaml blob in the `Instance` and `Binding` resources, unmarshal into `map[string]interface{}` for our go OSB library, which in turn serializes them into json in the request we send to the broker, accurate?